src/main/java/cz/cvut/fel/a4m36jee/airlines/controller/util/destination/DestinationLatJSFValidator.java

```diff
@@ -21,17 +21,16 @@
 
     @Override
     public void validate(FacesContext facesContext, UIComponent uiComponent, Object o) throws ValidatorException {
-        if(o != null){
-            if(o != null){
-                Double lat = (Double) o;
-                if(lat > 90) {
-                    Messages.add(FacesMessage.SEVERITY_ERROR, "createDestinationLat", "Invalid Latitude! Latitude is greater than allowable maximum of 90.");
-                }
-                if(lat < -90) {
-                    Messages.add(FacesMessage.SEVERITY_ERROR, "createDestinationLat", "Invalid Latitude! Latitude is less than allowable minimum of -90.");
-                }
-            }
+        if (o == null) {
```

Comment:
as with all validators, I would merge both "if" branches and produce common message, such as "Latitude must be between <-90;90>" or similiar in order to reduce code duplicity.